tests/system/large/ml/test_decomposition.py

```diff
@@ -163,3 +163,54 @@ def test_decomposition_configure_fit_load_none_component(
         in reloaded_model._bqml_model.model_name
     )
     assert reloaded_model.n_components == 7
+
+
+def test_decomposition_mf_configure_fit_load(
+    session, ratings_df_default_index, dataset_id
+):
+    model = decomposition.MatrixFactorization(
+        num_factors=6,
+        feedback_type="explicit",
+        user_col="user_id",
+        item_col="item_col",
+        rating_col="ratings",
+        l2_reg=9.83,
+    )
+    model.fit(
+        ratings_df_default_index.rename(
+            columns={"rating": "rating_col", "item_id": "item_col"}
+        )
+    )
+
+    reloaded_model = model.to_gbq(
+        f"{dataset_id}.temp_configured_mf_model", replace=True
+    )
+
+    new_ratings = session.read_pandas(
+        pd.DataFrame(
+            {
+                "user_id": ["11", "12", "13"],
+                "item_id": [1, 2, 3],
+                "ratings": [1.0, 2.0, 3.0],
+            }
+        )
+    )
+
+    reloaded_model.score(new_ratings)
+
+    result = reloaded_model.predict(
+        new_ratings.rename(columns={"item_id": "item_col"})
+    ).to_pandas()
+
+    assert reloaded_model._bqml_model is not None
+    assert (
+        f"{dataset_id}.temp_configured_mf_model"
+        in reloaded_model._bqml_model.model_name
+    )
+    assert result is not None
+    assert reloaded_model.feedback_type == "explicit"
+    assert reloaded_model.num_factors == 6
+    assert reloaded_model.user_col == "user_id"
+    assert reloaded_model.item_col == "item_col"
+    assert reloaded_model.rating_col == "rating"
```

Comment:
Should be `ratings` once the `@property` is implemented.

```suggestion
    assert reloaded_model.rating_col == "ratings"
```